detox/src/android/espressoapi/ViewActions.js

```diff
@@ -17,28 +25,60 @@ class ViewActions {
     };
   }
 
-  static clearText() {
+  static actionWithAssertions(viewAction) {
     return {
       target: {
         type: "Class",
         value: "android.support.test.espresso.action.ViewActions"
       },
-      method: "clearText",
-      args: []
+      method: "actionWithAssertions",
+      args: [viewAction]
     };
   }
 
-  static click() {
+  static clearText() {
     return {
       target: {
         type: "Class",
-        value: "com.wix.detox.espresso.DetoxViewActions"
+        value: "android.support.test.espresso.action.ViewActions"
       },
-      method: "click",
+      method: "clearText",
       args: []
     };
   }
 
+  static click() {
+    function click0() {
```

Comment:
Again here, why are there two clicks?
I see only one in UIDevice API

Reply:
There are two because the method is overloaded.